chain/penumbra/penumbra_client_node.go

```diff
@@ -0,0 +1,396 @@
+	custodyv1alpha1 "github.com/strangelove-ventures/interchaintest/v8/chain/penumbra/custody/v1alpha1"
+	viewv1alpha1 "github.com/strangelove-ventures/interchaintest/v8/chain/penumbra/view/v1alpha1"
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/strangelove-ventures/interchaintest/v8/internal/dockerutil"
+	"github.com/strangelove-ventures/interchaintest/v8/testutil"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type PenumbraClientNode struct {
+	log *zap.Logger
+
+	KeyName      string
+	Index        int
+	VolumeName   string
+	Chain        ibc.Chain
+	TestName     string
+	NetworkID    string
+	DockerClient *client.Client
+	Image        ibc.DockerImage
+
+	address    []byte
+	addrString string
+
+	containerLifecycle *dockerutil.ContainerLifecycle
+
+	// Set during StartContainer.
+	hostGRPCPort string
+}
+
+func NewClientNode(
+	ctx context.Context,
+	log *zap.Logger,
+	chain *PenumbraChain,
+	keyName string,
+	index int,
+	testName string,
+	image ibc.DockerImage,
+	dockerClient *client.Client,
+	networkID string,
+	address []byte,
+	addrString string,
+) (*PenumbraClientNode, error) {
+	p := &PenumbraClientNode{
+		log:          log,
+		KeyName:      keyName,
+		Index:        index,
+		Chain:        chain,
+		TestName:     testName,
+		Image:        image,
+		DockerClient: dockerClient,
+		NetworkID:    networkID,
+		address:      address,
+		addrString:   addrString,
+	}
+
+	p.containerLifecycle = dockerutil.NewContainerLifecycle(log, dockerClient, p.Name())
+
+	tv, err := dockerClient.VolumeCreate(ctx, volumetypes.CreateOptions{
+		Labels: map[string]string{
+			dockerutil.CleanupLabel:   testName,
+			dockerutil.NodeOwnerLabel: p.Name(),
+		},
+	})
+	if err != nil {
+		return nil, fmt.Errorf("creating pclientd volume: %w", err)
+	}
+	p.VolumeName = tv.Name
+	if err := dockerutil.SetVolumeOwner(ctx, dockerutil.VolumeOwnerOptions{
+		Log: log,
+
+		Client: dockerClient,
+
+		VolumeName: p.VolumeName,
+		ImageRef:   image.Ref(),
+		TestName:   testName,
+		UidGid:     image.UidGid,
+	}); err != nil {
+		return nil, fmt.Errorf("set pclientd volume owner: %w", err)
+	}
+
+	return p, nil
+}
+
+const (
+	pclientdPort = "8081/tcp"
+)
+
+var pclientdPorts = nat.PortSet{
+	nat.Port(pclientdPort): {},
+}
+
+// Name of the test node container
+func (p *PenumbraClientNode) Name() string {
+	return fmt.Sprintf("pclientd-%d-%s-%s-%s", p.Index, p.KeyName, p.Chain.Config().ChainID, p.TestName)
+}
+
+// the hostname of the test node container
+func (p *PenumbraClientNode) HostName() string {
+	return dockerutil.CondenseHostName(p.Name())
+}
+
+// Bind returns the home folder bind point for running the node
+func (p *PenumbraClientNode) Bind() []string {
+	return []string{fmt.Sprintf("%s:%s", p.VolumeName, p.HomeDir())}
+}
+
+func (p *PenumbraClientNode) HomeDir() string {
+	return "/home/heighliner"
```

Comment:
good question, the actual full nodes in the network require `/home/heighliner` because of how the filesystem is setup via the Dockerfile that is used to build the penumbra images [here](https://github.com/strangelove-ventures/heighliner/blob/15a4dbc94689dce71efb0368b8bcfa2052cce526/dockerfile/cargo/Dockerfile#L308) but i think this occurrence could be any arbitrary string because the `PenumbraClientNode` describes a client daemon that is speaking to a full node.

I'll double check that the test still passes with any arbitrary string as the home dir and if so i'll change this to `/home/pclientd` since any files generated in this directory should specifically be related to the penumbra client daemon